src/application-insights/azext_applicationinsights/_params.py

```diff
@@ -79,3 +79,48 @@ def load_arguments(self, _):
     with self.argument_context('monitor app-insights component linked-storage') as c:
         c.argument('storage_account_id', options_list=['--storage-account', '-s'], validator=validate_storage_account_name_or_id,
                    help='Name or ID of a linked storage account.')
+
+    with self.argument_context('monitor app-insights component continues-export list') as c:
+        c.argument('application', options_list=['--app', '-a'], id_part=None,
+                   help='GUID, app name, or fully-qualified Azure resource name of Application Insights component. The application GUID may be acquired from the API Access menu item on any Application Insights resource in the Azure portal. If using an application name, please specify resource group.')
+
+    with self.argument_context('monitor app-insights component continues-export create') as c:
+        c.argument('record_types', nargs='+',
+                   arg_type=get_enum_type(
+                       ['Requests', 'Event', 'Exceptions', 'Metrics', 'PageViews', 'PageViewPerformance', 'Rdd',
+                        'PerformanceCounters', 'Availability', 'Messages']),
+                   help='The document types to be exported, as comma separated values. Allowed values include \'Requests\', \'Event\', \'Exceptions\', \'Metrics\', \'PageViews\', \'PageViewPerformance\', \'Rdd\', \'PerformanceCounters\', \'Availability\', \'Messages\'.')
+        c.argument('dest_sub_id', help='The subscription ID of the destination storage account.')
```

Comment:
define a argument group for these arguments, for example 'arg_group="destination"', you can find other code for reference.